cross-chain-atomic-swap-cordapp/src/main/kotlin/jp/co/layerx/cordage/crosschainatomicswap/contract/ProposalContract.kt

```diff
@@ -36,12 +35,8 @@ open class ProposalContract: Contract {
                     (proposalCommand.signers.toSet() == proposal.participants.map { it.owningKey }.toSet())
             }
             is ProposalCommands.Consume -> requireThat {
```

Comment:
Isn't it necessary to verify whether the content of the ProposalState and the content of the Bond being transferred are the same here?

This is a validation that was previously written in SecurityContract.

Reply:
it slipped in my mind. I'll add the constraint.